app/backend/src/couchers/servicers/conversations.py

```diff
@@ -342,20 +342,32 @@ def SearchMessages(self, request, context):
             )
 
     def CreateGroupChat(self, request, context):
-        if len(request.recipient_user_ids) < 1:
+        with session_scope() as session:
+            recipient_user_ids = [
+                user.id
+                for user in (
+                    session.query(User.id).filter(User.is_visible).filter(User.id.in_(request.recipient_user_ids)).all()
+                )
+            ]
+
+        # make sure all requested users are visible
+        if len(recipient_user_ids) != len(request.recipient_user_ids):
+            context.abort(grpc.StatusCode.INVALID_ARGUMENT, errors.USER_NOT_FOUND)
```

Comment:
Combine this with the `set`:

`if len(set(recipient_user_ids)) != len(request.recipient_user_ids):`

Reply:
I wanted a different error code. User Not Found vs Invalid recipient